src/components/SplitPane.tsx

```diff
@@ -207,3 +249,47 @@ function getSize(
 
   return style;
 }
+
+interface SplitterProps {
+  onDoubleClick: () => void;
+  onMouseDown: (event: React.MouseEvent) => void;
+  onMouseUp: () => void;
+  orientation: SplitOrientation;
+  isFinalClosed: boolean;
+}
+
+function Splitter(props: SplitterProps) {
+  const { onDoubleClick, onMouseDown, onMouseUp, orientation, isFinalClosed } =
+    props;
+  return (
+    <div
+      onDoubleClick={onDoubleClick}
+      onMouseDown={onMouseDown}
+      onMouseUp={onMouseUp}
+      css={cssStyles.separator(orientation, !isFinalClosed)}
+    >
+      <div css={css({ fontSize: 10 })}>
+        {orientation === 'horizontal' ? <span>⋮</span> : <span>⋯</span>}
+      </div>
+    </div>
+  );
+}
+
+interface SplitPaneInnerProps {
+  style: CSSProperties;
+  value: boolean;
+  children: React.ReactNode;
+}
+
+const SplitPaneInner = forwardRef<HTMLDivElement, SplitPaneInnerProps>(
```

Comment:
I think we should avoid to overuse `forwardRef`, especially in internal components. You can pass the ref with a simple prop, for example `innerRef`, or `divRef`.

Reply:
Yes, I agree, you need it as internal props to be sure that it is updated. I did it like that because the component will re-render, for example, if we use the component in another component like NMRium.

But you are right, it is a general component, and will use it not only in NMRium , I will make the changes.